assets/python/server.py

```diff
@@ -1,23 +1,132 @@
+import os
+import sys
+import logging
+import ipaddress
+import http.client
+
 from http.server import HTTPServer, BaseHTTPRequestHandler
 from socketserver import ThreadingMixIn
-import os
+
+ENDPOINT_TYPE_MAP = {
+            'api.ipify.org': "IPv4",
+            'api6.ipify.org': "IPv6",
+            'api64.ipify.org': "Dual stack"
+        }
+
+PORT = '8080'
```

Comment:
maybe this could be called `DEFAULT_PORT`, as it is the fallback when there is no `PORT` env variable (e.g. when debugging locally)